project/.github/workflows/ci.yml.jinja

```diff
@@ -33,11 +33,15 @@ jobs:
 {% endif %}{% raw %}
   test:
     name: ${{ matrix.platform }} (${{ matrix.python-version }})
-    runs-on: ${{ matrix.platform }}
+    runs-on: ${{ matrix.platform }}{% endraw %}
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
+        python-version: [{%- if minimum_python <= 9 %}"3.9", {% endif -%}
+          {% if minimum_python <= 10 %}"3.10", {% endif -%}
+          {% if minimum_python <= 11 %}"3.11", {% endif -%}
+          {% if minimum_python <= 12 %}"3.12",  {% endif -%}
+          {% if minimum_python <= 13 %}"3.13"{% endif %}{% raw %}]
```

Comment:
I'm not sure what the jinja formatting best practices are for breaking up a statement into multiple lines while having it render as a single line. This is the best I could come up with - please feel free to suggest an alternative.